### ISSUE TITLE
names and intervals

### ISSUE THREAD

**Issue by peeviddy** [winterdelta/keplersmachines#1]
Is each observation per star evenly spaced timewise?

also, are there names for each of the stars in the dataset somewhere?

**Comment by winterdelta** [comment]
brilliant question. The second one that is. I tore some hair out trying to locate the label csv which stored the ref that NASA uses for each of the stars in this dataset. It remans elusive, in the wild. The story continues.

**Comment by RamseyV** [comment]
It seems as though each campaign lasted roughly 80 days. The telescope had 2 time settings: long cadence (30 min) and short cadence (1 min). "The long cadence was expected to yield about 10,000 to 20,000 targets with the short cadence only up to 100 targets" [Source 1], since it is a shorter exposure time, it can't detect as much light from distant stars. This would give about 3,800 flux values for exactly 80 days for the long cadence. With this and the size of the dataset I would assume that the data is of long cadence. The dataset doesn't seem complete, as there are more observation listed on the Milkulski archive for campaign 3, but it does seem to have all of the stars with exoplanets listed in the archive. 

TL;DR: Yes the flux values are evenly spaced timewise. 

Source 1 - https://keplerscience.arc.nasa.gov/k2-observing.html

**Comment by RamseyV** [comment]
For the second part of your question, it looks like this page has all of the targets from all of the K2 campaigns - https://keplerscience.arc.nasa.gov/k2-approved-programs.html.
